tools/oas-to-am/README.md

```diff
@@ -0,0 +1,30 @@
+# OpenAPI Specification to Assign Message
+
+A simple tool to generate an Assign Message policy from a JSON OpenAPI Spec.
+
+## Usage
+
+``` sh
+git clone https://github.com/apigee/devrel
+cd devrel/tools/oas-to-am
+export PATH=$PATH:$PWD/bin #you can add this to your .bashrc too
+SPEC=./path/to/spec.json OPERATION=operationId oas-to-am
+```
+
+## Dependencies
+
+- jq
+- libxml2-utils
+
+## Current Behaviour
+
+Takes the OAS spec [here](./test/features/fixtures/petstore.json), parses it to
+generate [this](./test/features/fixtures/expected.xml). The behaviour is
+described [here](./test/features/OASToAM.feature).
+
+## Purpose
```

Comment:
The main purpose IMO is to generate requests and flow callouts for backend systems based on the OAS. It is mentioned in the test and the template comment but should also be stated here in the readme.